sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -207,12 +207,8 @@ public virtual TokenCredential Credential
                 else
                 {
                     var clientSecret = GetOptionalVariable("CLIENT_SECRET");
-                    if (string.IsNullOrWhiteSpace(clientSecret))
-                    {
-                        _credential = new DefaultAzureCredential(
-                            new DefaultAzureCredentialOptions { ExcludeManagedIdentityCredential = true });
-                    }
-                    else
+                    var systemAccessToken = GetOptionalVariable("SYSTEM_ACCESSOTOKEN");
+                    if (!string.IsNullOrWhiteSpace(clientSecret))
```

Comment:
Do we want to give precedence to the client secret?   Should we use that as the fallback for the system token instead?

Reply:
SYSTEM_ACCESSTOKEN will be defined even if the test pipeline in question hasn't been switched over to use pipeline auth, this is why CLIENT_SECRET is evaluated first.